torch_xla/distributed/fsdp/utils.py

```diff
@@ -142,6 +143,13 @@ def _try_patching_forward_method(m, forward_method_name="forward"):
     if forward_method is None:
       return
     if getattr(forward_method, "__func__", None) != torch.nn.Linear.forward:
+      m_cls = m.__class__
+      if m_cls is not torch.nn.Linear and issubclass(m_cls, torch.nn.Linear):
+        logging.warning(
```

Comment:
Should this warning be generalized for all of Lightning when someone uses xla? To my understanding all Linear() Layers will have to go through this patch.